components/namespace-lister/production/base/kustomization.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- deployment.yaml
+- namespace.yaml
+- rbac.yaml
+- service.yaml
+- network_policy.yaml
+namespace: namespace-lister
+images:
+- name: namespace-lister
+  newName: quay.io/konflux-ci/namespace-lister
+  newTag: fd195c941b3151c165ddf376ce5f44d57db3f071
```

Comment:
This image is quite old, can we bump it?

Reply:
Fixed in 28bd12fe.